### PR TITLE
chore: add volunteer eskayML 

### PR DESCRIPTION
**Pull request by eskayML**
Added myself as a contributor

### CODE REVIEW

File: data/users/eskayML.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+
+  "name": "eskayML",
+
+  "bio": "Python Developer | Data Scientist ",
```
Comment (codewithvoid) [inline data/users/eskayML.json:5]: Better to pick 1. Preferably Data Scientist :)

File: data/users/eskayML.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  "image": "https://avatars.githubusercontent.com/u/71832907",
+
+  "categories": [
+
+    {
+
+      "slug": "programming | machine-learning | data-science",
```
Comment (codewithvoid) [inline data/users/eskayML.json:13]: just `data-science` slug is fine

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks @eskayML for raising PR. Added small comments. PTAL.

**Comment by eskayML** [comment]
Made the changes , thanks

**Comment by codewithvoid** [comment]
Thanks for making the changes @eskayML. You can join the community [here](https:/discord.gg/DFwsFKfgEB)

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**